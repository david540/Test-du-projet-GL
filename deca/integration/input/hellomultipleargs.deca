{
    println("Salut, j'ai ", 21, " ans et je mesure ", 1.83, " m");
}
