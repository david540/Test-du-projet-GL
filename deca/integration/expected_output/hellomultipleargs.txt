Salut, j'ai 21 ans et je mesure 1.83000e+00 m
